Use resize2fs to shrink system.img

ZeroPointEnergy opened the pull request:
This replaces the whole make_ext4fs and mount stuff with resize2fs
for shrinking the system.img like it is already done in the
replace-android-image script.

bhush9 commented:
re: sudo

```
21:42 <@bshah> the reason it works completely fine for you is because you have udev rules setup properly
21:42 <@bshah> lot of people don't
```

ZeroPointEnergy committed:
Use resize2fs to shrink system.img

This replaces the whole make_ext4fs and mount stuff with resize2fs
for shrinking the system.img like it is already done in the
replace-android-image script.

ZeroPointEnergy commented:
Now with more sudo

faizauthar12 commented:
and now android-tools has a new tools "mke2fs" and they drop make_ext4fs
